config.yaml

```diff
@@ -2,6 +2,7 @@ owner: netbox-community
 repo: netbox-chart
 remote: origin
 target-branch: develop
+timeout: 10m
```

Comment:
I don't think this will work. The structure should follow the args provided on the command line.
See the template I've put in my comment: https://github.com/netbox-community/netbox-chart/pull/237#discussion_r1626594828

Reply:
ohhhh, now I get it, need to do a `helm-extra-args: --timeout 10m` in here

Reply:
aha, no, it doesn't take a list like some of the others, but `ct lint` _does_ validate it for me. The magic incantation is `helm-extra-args: "--timeout 10m"`